xla/backends/cpu/benchmarks/e2e/gemma2/keras/benchmark.py

```diff
@@ -75,27 +88,34 @@ def run(gemma_lm, max_len):
   times = []
   for i in range(1, 6):
     start = time.time()
-    output = gemma_lm.generate(_QUERY, max_length=max_len + 1)
-    assert num_actual_output_tokens == len(output.split(" "))
+    output = gemma_lm.generate(_QUERY, max_length=total_output_len + 1)
     elapsed_time = (time.time() - start) * 1000
+    assert num_actual_output_tokens == len(tokenizer(output))
     times.append(elapsed_time)
+    gen_tokens = 0
+    if num_actual_output_tokens > in_tokens_len:
+      gen_tokens = num_actual_output_tokens - in_tokens_len
```

Comment:
We don't need these lines since both `num_actual_output_tokens` and `in_tokens_len` are unchanged. (The assert in line 93 made sure `num_actual_output_tokens` stays the same.)

Reply:
Shall we remove the assert instead? and i can make these lines concise as you suggested above. 
Because the generated output is not always same and num_actual_output_tokens can vary. It somehow works in this example with a small _NUM_OUTPUT_TOKENS = 30, but it doesn't when this number is changed.